src/persist-client/src/impl/machine.rs

```diff
@@ -997,6 +1016,7 @@ mod tests {
             .expect_open::<String, (), u64, i64>(ShardId::new())
             .await;
         let consensus = Arc::clone(&write.machine.consensus);
+        let now = SYSTEM_TIME.clone();
```

Comment:
we should use the one available as `write.cfg.now` instead (so the whole test has the same clock)